src/main/java/net/mcreator/element/converter/fv29/entities/ArmorTexturesConverter.java

```diff
@@ -0,0 +1,75 @@
+/*
+ * MCreator (https://mcreator.net/)
+ * Copyright (C) 2012-2020, Pylo
+ * Copyright (C) 2020-2022, Pylo, opensource contributors
+ *
+ * This program is free software: you can redistribute it and/or modify
+ * it under the terms of the GNU General Public License as published by
+ * the Free Software Foundation, either version 3 of the License, or
+ * (at your option) any later version.
+ *
+ * This program is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ * GNU General Public License for more details.
+ *
+ * You should have received a copy of the GNU General Public License
+ * along with this program.  If not, see <https://www.gnu.org/licenses/>.
+ */
+
+
+package net.mcreator.element.converter.fv29.entities;
+
+import com.google.gson.JsonElement;
+import net.mcreator.element.GeneratableElement;
+import net.mcreator.element.converter.IConverter;
+import net.mcreator.element.types.Armor;
+import net.mcreator.io.FileIO;
+import net.mcreator.ui.workspace.resources.TextureType;
+import net.mcreator.util.FilenameUtilsPatched;
+import net.mcreator.workspace.Workspace;
+
+public class ArmorTexturesConverter implements IConverter {
+	@Override
+	public GeneratableElement convert(Workspace workspace, GeneratableElement input, JsonElement jsonElementInput) {
+		Armor armor = (Armor) input;
+
+		if (armor.helmetModelTexture != null && !armor.helmetModelTexture.isEmpty()) {
+			FileIO.copyFile(workspace.getFolderManager()
+							.getTextureFile(FilenameUtilsPatched.removeExtension(armor.helmetModelTexture), TextureType.OTHER),

```

Comment:
Armor textures seem to have and use their own texture type/directory, so I don't understand why are you moving armor pictures from other to entity textures.